src/content/blog/ddev-jan-2025-newsletter.md

```diff
@@ -0,0 +1,44 @@
+
+**DDEV Notes and News**
+
+- [DDEV Advisory Group Meeting](https://github.com/orgs/ddev/discussions/6682). This month's meeting is Wednesday, January 8, 2025, and is a critical discussion of our financial situation, budget, and plans based on recent news [join us](https://www.meetup.com/ddev-events/events/303197425). If you'd like a calendar invitation instead of the meetup signup, just reply to this email.
+- [DDEV, solr, and platform.sh](https://www.computerminds.co.uk/articles/ddev-solr-and-platformsh)
+- [Joshuami blog on Drupal CMS (via DDEV)](https://joshuami.com/blog/2024-12/blog-drupal-cms-starshot-kinda-now/)
+- [Testing Drupal 11 with Cypress and DDEV](https://www.thedroptimes.com/44845/how-test-drupal-11-using-cypress-and-ddev-drupaladicto-explains) shows how to use [@tyler36](https://github.com/tyler36)'s excellent [Cypress add-on for DDEV](https://github.com/tyler36/ddev-cypress).
+- [Contao CMS docs show preferred DDEV approach](https://docs.contao.org/manual/en/guides/local-installation/ddev/)
+- DDEV Releases and announcements are now made on [Bluesky](https://bsky.app/profile/ddev.bsky.social) as well as other social networks, but no longer on X/Twitter.
+
+**Funding DDEV and especially Maintainer Stas**: _We need your help and your organization's help! Let me know if you need help getting this into your 2025 budget!_ Our key financial goal is to [fully fund @stasadev so he can work exclusively on DDEV](lets-fund-stas-maintainer.md). We've slipped back in recent months. We need about $3700/month in increased pledges from organizations and individuals. See [Full information about supporting DDEV](https://github.com/sponsors/ddev). We’re happy to invoice you, happy to [do a call discussing this](https://cal.com/randyfay/30min), and would love it if you’d include DDEV in your 2025 budgeting process. (Our current status: We receive about $3000/month, have been spending about $3000-$4000/month. Bank balance is about $8683 down from $9,300 last month.)
+
+**THANKS to all of you who are supporting DDEV’s path to sustainability** and who have gotten your organizations to do so.
+
+Want to keep up as the month goes along? Follow on
+
+- [blog](https://ddev.com/blog/)
+- [LinkedIn](https://www.linkedin.com/company/ddev-foundation)
+- [Mastodon](https://fosstodon.org/@ddev)
+- [Bluesky](https://bsky.app/profile/ddev.bsky.social)
+- and join our community on [Discord](/s/discord)
+
+Happy December from Randy Fay, DDEV Project Lead.
```

Comment:
```suggestion
Happy January from Randy Fay, DDEV Project Lead.
```

Seems strange to say Decemeber when the newsletter pubdate is January.

Reply:
Oops, thanks. Copy pasta.